src/Microsoft.DotNet.Interactive.PowerShell/Commands/OutDisplayCommand.cs

```diff
@@ -53,8 +55,17 @@ public sealed class OutDisplayCommand : PSCmdlet
     protected override void ProcessRecord()
     {
         object obj = InputObject is PSObject psObj ? psObj.Unwrap() : InputObject;
-        DisplayedValue displayedValue = display(obj, MimeType);
-            
+
+        DisplayedValue displayedValue = default;
+
+        if (ParameterSetName == "MimeType"){
+            displayedValue = display(obj, MimeType);
+        }
+
+        if (ParameterSetName == "CustomMimeType"){
```

Comment:
If we completely remove the `CustomMimeType` property and set `MimeType` in the test (or in the notebook for that matter), does it work as expected?